build/CoreGraphics/dll/CoreGraphics.vcxproj

```diff
@@ -77,7 +77,7 @@
     </ClCompile>
     <Link>
       <ModuleDefinitionFile>CoreGraphics.def</ModuleDefinitionFile>
-      <AdditionalDependencies>cairo.lib;pixman.lib;freetype.lib;libpng.lib;libz.lib;libjpeg.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>cairo.lib;pixman.lib;freetype.lib;libpng.lib;libz.lib;libjpeg.lib;libdispatch.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
need it for CGFontGetTypeID, in line with CF patterns

---
In reply to: [83330701](https://github.com/Microsoft/WinObjC/pull/1145#discussion_r83330701) [](ancestors = 83330701)